doc/source/Resources/PyAEDTInstallerFromDesktop.py

```diff
@@ -10,18 +10,25 @@
 
 
 def run_pyinstaller_from_c_python(oDesktop):
-    # This is called when run from IronPython
+    # Iron Python script to create the virtual environment and install PyAEDT
+    # Get AEDT information
     version = oDesktop.GetVersion()[2:6].replace(".", "")
+    # From AEDT 2023.2 the installed CPython version is 3.10
     python_version = "3.10"
     if version <= "231":
         python_version = "3.7"
```

Comment:
```suggestion
    python_version = "3.10" if version > "231" else "3.7"
```
This should be working in IronPython